plugins/progress/progress.plugin.sh

```diff
@@ -29,32 +31,57 @@ function delay()
+    if [ $_omb_plugin_progress_value -le 5 -a $value -ge 5 ]  ; then echo -ne "[#-------------------------] (5%)  $message$whitespace \r"  ; delay; fi;
+    if [ $_omb_plugin_progress_value -le 10 -a $value -ge 10 ]; then echo -ne "[##------------------------] (10%) $message$whitespace \r"  ; delay; fi;
+    if [ $_omb_plugin_progress_value -le 15 -a $value -ge 15 ]; then echo -ne "[###-----------------------] (15%) $message$whitespace \r"  ; delay; fi;
+    if [ $_omb_plugin_progress_value -le 20 -a $value -ge 20 ]; then echo -ne "[####----------------------] (20%) $message$whitespace \r"  ; delay; fi;
+    if [ $_omb_plugin_progress_value -le 25 -a $value -ge 25 ]; then echo -ne "[#####---------------------] (25%) $message$whitespace \r"  ; delay; fi;
+    if [ $_omb_plugin_progress_value -le 30 -a $value -ge 30 ]; then echo -ne "[######--------------------] (30%) $message$whitespace \r"  ; delay; fi;
+    if [ $_omb_plugin_progress_value -le 35 -a $value -ge 35 ]; then echo -ne "[#######-------------------] (35%) $message$whitespace \r"  ; delay; fi;
+    if [ $_omb_plugin_progress_value -le 40 -a $value -ge 40 ]; then echo -ne "[########------------------] (40%) $message$whitespace \r"  ; delay; fi;
+    if [ $_omb_plugin_progress_value -le 45 -a $value -ge 45 ]; then echo -ne "[#########-----------------] (45%) $message$whitespace \r"  ; delay; fi;
+    if [ $_omb_plugin_progress_value -le 50 -a $value -ge 50 ]; then echo -ne "[##########----------------] (50%) $message$whitespace \r"  ; delay; fi;
+    if [ $_omb_plugin_progress_value -le 55 -a $value -ge 55 ]; then echo -ne "[###########---------------] (55%) $message$whitespace \r"  ; delay; fi;
+    if [ $_omb_plugin_progress_value -le 60 -a $value -ge 60 ]; then echo -ne "[############--------------] (60%) $message$whitespace \r"  ; delay; fi;
+    if [ $_omb_plugin_progress_value -le 65 -a $value -ge 65 ]; then echo -ne "[#############-------------] (65%) $message$whitespace \r"  ; delay; fi;
+    if [ $_omb_plugin_progress_value -le 70 -a $value -ge 70 ]; then echo -ne "[###############-----------] (70%) $message$whitespace \r"  ; delay; fi;
+    if [ $_omb_plugin_progress_value -le 75 -a $value -ge 75 ]; then echo -ne "[#################---------] (75%) $message$whitespace \r"  ; delay; fi;
+    if [ $_omb_plugin_progress_value -le 80 -a $value -ge 80 ]; then echo -ne "[####################------] (80%) $message$whitespace \r"  ; delay; fi;
+    if [ $_omb_plugin_progress_value -le 85 -a $value -ge 85 ]; then echo -ne "[#######################---] (90%) $message$whitespace \r"  ; delay; fi;
+    if [ $_omb_plugin_progress_value -le 90 -a $value -ge 90 ]; then echo -ne "[##########################] (100%) $message$whitespace \r" ; delay; fi;
+    if [ $_omb_plugin_progress_value -le 100 -a $value -ge 100 ];then echo -ne 'Done!                                            \n' ; delay; fi;
```

Comment:
Regarding `printf`, I was getting `bash: [: too many arguments` with printf. I believe this broke last month. We should likely switch to using `_omb_util_print` instead.

Reply:
Ah, yeah. `(0%)` should be `(0%%)` in the printf format.